tweets/2021-07/telemetry.tweet

```diff
@@ -0,0 +1 @@
+Analyzing operational data collected from OpenShift CI clusters can help provide insights into OpenShift releases. Watch https://youtu.be/D38gR-7c2dc to learn how we use python to fetch prometheus metrics collected from CI clusters, and explore it in a jupyter notebook.
```

Comment:
@chauhankaranraj how about tagging the accounts?

```suggestion
Analyzing operational data collected from @OpenShift CI clusters can help provide insights into @OpenShift releases. Watch https://youtu.be/D38gR-7c2dc to learn how we use python to fetch Prometheus metrics collected from CI clusters, and explore it in a Jupyter notebook.
```

Reply:
I like that idea :) Should we "@ mention" (or maybe hashtag?) prometheus and jupyter too, or just openshift?

Reply:
sure, why not

Reply:
updated